CONTRIBUTING.md

```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
+
+Install Xcode and Homebrew Mac. Once that is done: 
```

Comment:
I googled it and it seems that a gcc compiler is needed for homebrew and pip, and xcode is one way to get that but not the only one. What do you suggest I put?